src/compiler/codegen_c_json_parser.c

```diff
@@ -1477,7 +1477,7 @@ static int gen_table_parser(fb_output_t *out, fb_compound_type_t *ct)
     println(out, "");
     println(out, "static inline int %s_parse_json_as_root(flatcc_builder_t *B, flatcc_json_parser_t *ctx, const void *buf, size_t bufsiz, int flags, const char *fid)", snt.text);
     println(out, "{"); indent();
-    println(out, "return flatcc_json_parser_table_as_root(B, ctx, buf, bufsiz, flags, fid, %s_parse_json_table);",
+    println(out, "return flatcc_json_parser_table_as_root(B, ctx, (char*)buf, bufsiz, flags, fid, %s_parse_json_table);",
             snt.text);
```

Comment:
It's the `buf` argument that is wrong. It should be const char * not const void *. Note that the prototype is already generated earlier in the file.

Reply:
Right. But there was no prototype, my mistake, nor is one needed. But you found the dead code for the struct parser. It should actually be enabled. I'll merge this and figure it out why the code was dead. I'll be good if you can test again when I get to it, I'm on it.